radix-engine-tests/assets/wasm/loop_v2.wat

```diff
@@ -0,0 +1,29 @@
+(module
+
+  ;; Simple function that always returns `()`
+  (func $Test_f (param $0 i64) (result i64)
+    ;; Loop starts!
+    (loop $loop
+      br $loop
+    )
+    ;; Loop ends!
+
+    ;; Encode () in SBOR at address 0x0
```

Comment:
All this stuff isn't actually needed because the loop spins forever, right? Or is some return needed for the thing to pass validation?

Reply:
It's not required in this case, as we don't care about the execution results. It has little impact on the test.

Reply:
Sounds good. Happy with your choice of leave it / comment it / delete it.